### PR TITLE
Fix DiffTool DatePicker translation label

### PR DESCRIPTION
**Pull request by nf-s**
### Fix DiffTool DatePicker translation label

Fixes https://github.com/TerriaJS/terriajs/issues/5236

Before we had DatePickers A and B use same translation key: `dateComparisonA`, where as DatePicker B should have been using `dateComparisonB`

image

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Comment by mwu2018** [comment]
Verified via http://ci.terria.io/fix-difftool-date-label/#share=s-dqXe6xHERG6JwMa98AmSRJ18xEO

File: CHANGES.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Change Log
 * Fix hard-coded colour value in Story
 * Use `configParameters.cesiumIonAccessToken` in `IonImageryCatalogItem`
 * Added support for skipping comments in CSV files
+* Fix `DiffTool` date-picker label `dateComparisonB`
```
Comment (mwu2018) [inline CHANGES.md:44]: There is merge conflict here.

**Review by mwu2018 — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (nf-s) [reply CHANGES.md:44]: Done!

**Review by mwu2018 — APPROVED** [review]

**Comment by nf-s** [comment]
Thanks @mwu2018 !

**Pull request merged**